.profile.d/heroku-metrics-daemon.sh

```diff
@@ -5,63 +5,57 @@ if [[ -z "$HEROKU_METRICS_URL" ]] || [[ "${DYNO}" = run\.* ]]; then
     return 0
 fi
 
-export HEROKU_METRICS_PROM_ENDPOINT=${HEROKU_METRICS_PROM_ENDPOINT:-/metrics}
-export HEROKU_METRICS_PROM_PORT=$((PORT + 1))
+# heroku-metrics-agent.jar is added in bin/compile
+if [[ -f ${HOME}/bin/heroku-metrics-agent.jar ]]; then
+    if [[ -f build.sbt ]] || # Scala
+       [[ -d target/resolution-cache ]]; then # Scala (sbt-heroku)
+        export JAVA_OPTS="-javaagent:${HOME}/bin/heroku-metrics-agent.jar ${JAVA_OPTS}"
+    else
+        export JAVA_TOOL_OPTIONS="-javaagent:${HOME}/bin/heroku-metrics-agent.jar ${JAVA_TOOL_OPTIONS}"
+    fi
+else
+    export HEROKU_METRICS_PROM_ENDPOINT=${HEROKU_METRICS_PROM_ENDPOINT:-/metrics}
```

Comment:
These aren't needed for ruby, as it uses statsd.

Reply:
do you want me to kill them? it's fine for java

Reply:
Fixed in #30